power_supplies:
  name: ['/motors_battery','/PSU_battery']
  max_voltage: [24.0,24.0]
range_sensors:
  name: ['/sensor/left_sonar','/sensor/right_sonar', 'default']
  frame_id: ['/left_sonar', '/right_sonar', '']
  radiation_type: [0,0,0]
  field_of_view: [80.0,80.0,80.0]
  min_range: [0.2,0.2,0.2]
  max_range: [4.5,4.5,4.5]
